routers/bots.py

```
@@ -0,0 +1,250 @@
+        return JSONResponse(content={"detail": f"Bot has been disabled."}, status_code = 200)
+
+    except HTTPException as e:
+            raise e
+    except Exception as e:
+        raise HTTPException(status_code = 500, detail = f"An error occurred: {str(e)}")
+    
+@bots_router.post('/update')
+@x_super_team
+@x_app_key
+@jwt_token
+async def enable(request: Request):
```

Ruff (0.8.2)

Redefinition of unused `enable` from line 178

(F811)